src/components/SingleArticle.jsx

```diff
@@ -29,22 +29,25 @@ export const SingleArticle = () => {
 	return isError ? (
 		<p className="error-display">There has been an error!</p>
 	) : isLoading ? (
-		<p className="loading-display">Loading Articles!</p>
+		<p className="loading-display">Loading Article!</p>
 	) : (
 		<>
 			<section className="single-article">
 				<div className="single-article-headings">
 					<h2>{article.title}</h2>
 					<p className="single-article-author">{article.author}</p>
 				</div>
-				<div>
+				<div className="article-main">
 					<img src={article.article_img_url} alt={article.title} />
-				</div>
-				<div>
 					<p className="single-article-body">{article.body}</p>
 				</div>
 			</section>
-			<section></section>
+			<section className="article-comments">
+				<h2>Comments</h2>
+				{allComments.map((comment) => {
```

Comment:
is there anything you could render when there are no comments? Have a look at `/articles/37`